src/wasm.h

```diff
@@ -954,6 +954,9 @@ class Module {
 
   void updateMaps();
 
+  // returns true if the module has a target features section
```

Comment:
Perhaps another option is an `std::pair<FeatureSet, bool>` like the map etc. insert APIs? Or maybe just
```
// Read the features section into the out param, if it is present (if not, returns false).
bool readFeatureSection(FeatureSet&) const;
```
(i.e. change name + add commenting)
```